account-kit/infra/src/client/viem.ts

```diff
@@ -0,0 +1,33 @@
+import { createBundlerClient } from "viem/account-abstraction";
+import type { AlchemySmartAccountClientConfig } from "./smartAccountClient";
+
+// Yea this isn't great or going to work well, we should just keep using our smart account client, but change the account type to match the viem one
+export const foo = (params: AlchemySmartAccountClientConfig) => {
+  const cl = createBundlerClient({
+    transport: params.transport,
+    chain: params.chain,
+    paymasterContext: { policyId: params.policyId },
+    paymaster: {
+      async getPaymasterStubData(parameters) {
+        if (!("policyId" in parameters.context)) {
+          throw new Error("policyId is required");
+        }
+
+        throw new Error(
+          "this is the biggest issue with viem's approach. we can't skip gas estimation during paymaster flows..."
```

Comment:
💬 If we do some hacky stuff, like stub it out, it might be possible. But would is it worth the cost to our souls.
Is there a line of communication to the viem team on this at least?